config/kafka-capacity-config.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 # see https://docs.google.com/spreadsheets/d/1W4H1IgDXKv24Sf7aWeqeAqxjKVj8EJB9giYN9obqPgU/edit#gid=493008265
-ingressEgressThroughputPerSec: "2Mi"
+ingressThroughputPerSec: "2Mi"
+egressThroughputPerSec: "2Mi"
```

Comment:
We don't really need to change this right now since it's going to be removed later right? Depending on when this and the removal of this config lands on main, we may need to update the saas template twice?